### ISSUE TITLE
modify setup request

### ISSUE THREAD

**Issue by philolsonjr** [nishant-njs/mold_maintenance_db#94]
exir button must work as summit

**Comment by nishant-njs** [comment]
You asked us to add an exit button so the operator is redirected to Main or Master Switchboard depending on where they came from. If we associate to the Submit button operator will have to click the menu on the top to navigate to the switchboard which you didn’t want earlier since the submit button will check for mandatory fields and error out.

Having a Submit button is useful because the operator will see the error/successful message from the server. Exit button can be used to exit the form.

**Comment by philolsonjr** [comment]
Let me try again, you need to understand that we do not have computer savvy operators on the shop floor. I have spent the last year using various methods to :1., Make sure they entered all the data., 2., That the forms are very easy to use., 3., Some forms open with the select list that appears when the form opens. (this will be logged once the form and reports flow as they should)

You need to provide the same flow as Access, so the form exit button needs to the data checking (as in access) before exiting the form, and then exit the form. What can this not be done? If we cannot provide the flow the IT help desk will be have too many calls each days as to why the data is missing. 

What is the issue from your end??

**Comment by nishant-njs** [comment]
Exit button updated. Please test and close the issue if resolved.
